.doc_gen/metadata/route53_metadata.yaml

```diff
@@ -0,0 +1,35 @@
+# zexi 0.4.3
```

Comment:
If you really meant to use the same snippet for both examples, I'd cut this to just the ListHostedZones example.

Reply:
Ok